### PR TITLE
Update tests to show new share behavior.

### PR DESCRIPTION
**Pull request by coltfred**
This updates tests which currently only pass against dev.

### CODE REVIEW

File: tests/document_ops.rs [hunk 1]
```diff
@@ -15,13 +15,18 @@ async fn doc_list() -> Result<(), IronOxideErr> {
     let sdk = initialize_sdk().await?;
     let (other_user, _) = init_sdk_get_user().await;
     let doc = "secret".to_string().into_bytes();
-    // grant_to_author is false, but doc should still come back in document_list
+    // grant_to_author is false, but doc should not come back in document list
```
Comment (giarc3) [inline tests/document_ops.rs:18]: A nitpick, but I think it's important for clarity
```suggestion
    // grant_to_author is false, so doc should not come back in document list
```

File: tests/document_ops.rs [hunk 2]
```diff
@@ -508,6 +513,43 @@ async fn doc_create_without_self_grant() -> Result<(), IronOxideErr> {
...
+                Some(create_id_all_classes("").try_into()?),
+                Some("first name".try_into()?),
+                true,
+                vec![UserOrGroup::User { id: user2.clone() }],
+            ),
+        )
+        .await?;
+
+    // should be a user with access, but not the currently initd user
+    assert_eq!(doc_result.grants().len(), 2);
```
Comment (giarc3) [inline tests/document_ops.rs:536]: What does this comment mean?

File: tests/document_ops.rs [hunk 3]
```diff
@@ -508,6 +513,43 @@ async fn doc_create_without_self_grant() -> Result<(), IronOxideErr> {
...
+    //Revoke the creator, which should now be allowed.
+    let revoke_result = user2_sdk
+        .document_revoke_access(
+            doc_result.id(),
+            &vec![UserOrGroup::User { id: user1.clone() }][..],
+        )
+        .await?;
```
Comment (giarc3) [inline tests/document_ops.rs:543]:
```suggestion
    let revoke_result = user2_sdk
        .document_revoke_access(doc_result.id(), &[UserOrGroup::User { id: user1.clone() }])
        .await?;
```

**Review by giarc3 — APPROVED** [review]

File: tests/document_ops.rs (hunk 2 above)
Comment (coltfred) [reply tests/document_ops.rs]: It's left over from the base test. Removed.

**Pull request merged**